Common/src/config_structure.cpp

```diff
@@ -2947,6 +2947,12 @@ void CConfig::SetPostprocessing(unsigned short val_software, unsigned short val_
     
   }
   
+  /*--- Check for Boundary condition option agreement ---*/
+
+  if ((Kind_Solver == NAVIER_STOKES || Kind_Solver == RANS) && Reynolds <=0){
```

Comment:
Could you add a further condition that the error will only be thrown for compressible cases?

In case of incompressible fluids, the Reynolds is derived from other quantities.

Reply:
Done!